zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -681,7 +681,7 @@ public String getStrictTransport() {
     ZEPPELIN_SERVER_XFRAME_OPTIONS("zeppelin.server.xframe.options", "SAMEORIGIN"),
     ZEPPELIN_SERVER_JETTY_NAME("zeppelin.server.jetty.name", null),
     ZEPPELIN_SERVER_STRICT_TRANSPORT("zeppelin.server.strict.transport", "max-age=631138519"),
-    ZEPPELIN_SERVER_X_XSS_PROTECTION("zeppelin.server.xxss.protection", 1);
+    ZEPPELIN_SERVER_X_XSS_PROTECTION("zeppelin.server.xxss.protection", "1");
```

Comment:
@felixcheung That's intentional.  When "X-XSS-Protection" header when value is set to "1", it enables XSS filtering (usually default in browsers). If a cross-site scripting attack is detected, the browser will sanitize the page (remove the unsafe parts). 

Ideally, it should be set to "X-XSS-Protection:1; mode=block". With this value, rather than sanitizing the page, the browser will prevent rendering of the page if an attack is detected. 

Hence, the default value is set to "1" so that it should not become strict. Those who want this security feature can enable it from Config, where putting "1" doesn't change anything and hence the ideal value "1; mode=block". If you feel, I should change the config value for sake of conformity across project, do let me know.